server/py/services/alerts/tests/unit/api/test_alerts.py

```diff
@@ -30,6 +30,69 @@ def test_store_alerts(self, db: Session, client: TestClient):
         project = "test-alerts"
         alert_name = "alert-name"
         self._create_project(db, project)
+        alert_config = self._create_alert_config(alert_name, project)
```

Comment:
why not use here the generate_alert_data that we have in `alerts/tests/unit/crud/utils.py` ?
I think that it is good that we start reduce these duplications as part of the ongoing refactoring of the alerts code.